lib/events/test/suite.go

```diff
@@ -288,15 +318,15 @@ func (s *EventsSuite) SessionEventsCRUD(t *testing.T) {
 		}}
 	}
 
-	history, _, err = s.Log.SearchSessionEvents(s.Clock.Now().Add(-1*time.Hour), s.Clock.Now().Add(2*time.Hour), 100, types.EventOrderAscending, "", withParticipant("alice"), "")
+	history, _, err = s.Log.SearchSessionEvents(s.Clock.Now().UTC().Add(-1*time.Hour), s.Clock.Now().UTC().Add(2*time.Hour), 100, types.EventOrderAscending, "", withParticipant("alice"), "")
```

Comment:
just adding missing UTC in test cases, as mention on auditlogger intefrace, timerange is defined as fromUTC and toUTC. In athenalogger I have assumed that those values will be in UTC, that's why those changes are required. I can also change athena implementation and enforce UTC there and revert those changes here. WDYT @rosstimothy